course/templates/course/index.html

```diff
@@ -12,85 +12,148 @@ <h1>{{ welcome_text|safe }}</h1>
 </div>
 {% endif %}
 
-<div class="row course-thumbnails">
-    {% site_advert as advert %}
-    {% if advert %}
+{% if my_instances %}
+<div class="frontpage-panel">
+    <h2 class="panel-heading">{% trans "My courses" %}</h2>
+    <div class="row frontpage">
+        {% for instance in my_instances %}
+        {% with instance_url=instance|url %}
         <div class="col-sm-6 col-md-4 col-lg-3">
-            <div class="thumbnail advert-thumbnail">
-                <div class="image" aria-hidden="true">
-                    {% if advert.image %}
-                        <img src="{{ advert.image }}" alt="">
-                    {% else %}
-                        <h1 class="glyphicon glyphicon-comment"></h1>
-                    {% endif %}
+            <div class="frontpage-course card">
+                <a href="{{ instance_url }}">
+                    <div class="course-image" aria-hidden="true">
+                        {% if instance.image %}
+                        <img class="card-img-top" src="{{ instance.image.url }}" alt="" />
+                        {% else %}
+                        <div class="card-img-top">
+                            <i class="glyphicon glyphicon-book"></i>
+                        </div>
+                        {% endif %}
+                    </div>
+                </a>
+                <div class="card-body">
+                    <h3 class="card-title" lang="{{ instance.language }}">
+                        <a href="{{ instance_url }}">
+                            {{ instance.course.name|parse_localization }}<br />
+                        </a>
+                    </h3>
+                    <h4 class="card-subtitle" lang="{{ instance.language }}">
+                        {{ instance.instance_name|parse_localization }}
+                    </h4>
+                    <p class="card-text">
+                        {{ instance.course.code }}
+                        {% if not instance.visible_to_students %}
+                        <span class="label label-danger">{% trans "Hidden from students" %}</span>
+                        {% endif %}
+                        <br />
+                        {{ instance.starting_time|date:"SHORT_DATE_FORMAT" }} &ndash;
+                        {{ instance.ending_time|date:"SHORT_DATE_FORMAT" }}
```

Comment:
This PR may be too specific for this issue, but in my opinion, we should not be changing the date format by language, nor should we be using the American date format at all. 

Would you like to look at that, or create a new issue somewhere else?

Reply:
Please create issue about that. The django date formatting is used in multiple places, so I would prefer we fix all of them to be consistent.

Reply:
Created #572

Reply:
Great, that'll get fixed in a separate place then